README.md

```diff
@@ -9,7 +9,7 @@
 * Gradle
 ```
 dependencies {
-    implementation 'com.github.backtrace-labs.backtrace-log4j2:backtrace-log4j2:0.9.1'
+    implementation 'com.github.backtrace-labs.backtrace-log4j2:backtrace-log4j2:0.9.4'
```

Comment:
are you sure we should bump the version to 0.9.4?

Reply:
let's keep it in the same version as backtrace-java